frontend/packages/core/src/AppLayout/user.tsx

```diff
@@ -76,34 +128,52 @@ const UserInformation: React.FC = () => {
   }
 
   return (
-    <Grid container alignItems="center" justify="flex-end">
-      <div>{userId()}</div>
+    <Box>
       <UserPhoto
         ref={anchorRef}
         edge="end"
         aria-controls={open ? "account-options" : undefined}
         aria-haspopup="true"
         onClick={handleToggle}
       >
-        <Avatar>
-          <Initials variant="h6">{userId().slice(0, 2).toUpperCase()}</Initials>
-        </Avatar>
+        <UserAvatar />
       </UserPhoto>
-      <Popper open={open} anchorEl={anchorRef.current} role={undefined} transition>
+      <Popper
```

Comment:
We might want to look into using popover for the user menu. Blocking scrolling and clicks might be reasonable here. Definitely investigate this later though

Reply:
okay! will leave a to-do to investigate